inscriptions/ISic1636.xml

```diff
@@ -170,18 +175,28 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#VM">
+                <ab>
+                    <lb n="1"/><g type="staurogram">&#65952;</g> Hic iacet be<hi rend="ligature">ne</hi><hi rend="ligature">me</hi>
+                    <lb n="2" break="no"/>morie A<hi rend="supraline">l</hi>b<hi rend="supraline">ina</hi> <expan><abbr><hi rend="supraline">h</hi></abbr><ex>onesta</ex></expan> <expan><abbr><hi rend="supraline">f</hi></abbr><ex>emina</ex></expan> <hi rend="supraline">qui</hi>
+                    <lb n="3"/><expan><abbr>vixs</abbr><ex>it</ex></expan> <expan><abbr><hi rend="supraline">an</hi></abbr><ex>nos</ex></expan> <expan><abbr><hi rend="supraline">pl</hi></abbr><ex>us</ex></expan> <expan><abbr><hi rend="supraline">m</hi></abbr><ex>inus</ex></expan> <num value="60">L<hi rend="stacked">X</hi></num> <hi rend="ligature">et</hi> requi
```

Comment:
Do we flag the s in vixit as reg/orig?

Reply:
I put `<expan><abbr>vi<choice><reg>x</reg><orig>xs</orig></choice></abbr><ex>it</ex></expan>`. tell me if it's ok and I will commit this last group

Reply:
Yes, looks good. Go ahead.